js/time.js

```diff
@@ -73,9 +75,22 @@ export const dateTimeString = (
     separator = "_",
     dateSeparator = "/",
     timeSeparator = ":",
-    { year = true, month = true, day = true, hours = true, minutes = true, seconds = true } = {}
+    {
+        year = true,
+        month = true,
+        day = true,
+        reverse = false,
+        hours = true,
+        minutes = true,
+        seconds = true
+    } = {}

```

Comment:
Shouldnt this reverse reverse everything? Hours included?

Reply:
I believe not. It's common to have the date reversed, but I've never seen the time